src/classes/HH_NameSpec_TEST.cls

```diff
@@ -239,29 +346,26 @@ private with sharing class HH_NameSpec_TEST {
         HH Name, Formal Greeting, and Informal Greeting are correct
     **********************************************************************************************************/            
     static testMethod void ThreeContactsTwoLastnamesCustomFields() {
-        if (strTestOnly != '*' && strTestOnly != 'ThreeContactsTwoLastnamesCustomFields') return;
-                
-        list<Contact> listCon = new list<Contact>();
-        Contact c1 = new Contact (
-            Salutation = 'Skipper',
-            FirstName = 'John',
-            LastName = 'Doe',
-            npe01__WorkEmail__c = 'john@doe.com'
-            
-        );
-        listCon.add(c1);
-        Contact c2 = new Contact (
-            Salutation = 'Mrs.',
-            FirstName = 'Jane',
-            LastName = 'Doe',
-            npe01__WorkEmail__c = 'jane@doe.com'
-        );
-        listCon.add(c2);
-        Contact c3 = new Contact (
-            FirstName = 'Sally',
-            LastName = 'Smith'
-        );
-        listCon.add(c3);
+        if (strTestOnly != '*' && strTestOnly != 'ThreeContactsTwoLastnamesCustomFields') return;                
+        
+        Contact[] listCon = new Contact[] {
```

Comment:
Fixed strTestOnly assignment for new test methods. Please advise if this check should be removed for all test methods in this class.